src/app/components/shared/Helpers/helpers.ts

```diff
@@ -1 +1,3 @@
 export const capitalizeString = (s: string) => s.charAt(0).toUpperCase() + s.slice(1)
+
+export const appendTimeToDate = (date: string) => `${ date }T02:00:01.001+0200`
```

Comment:
Back-end says no.. But this is the correct way. But since the user can only select a date there is no point to set the time around midnight (with a change that the day before is selected because of the timezone). Therefore I set the time to midday (`T12:00:00+0200`)